engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -176,6 +176,13 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
             // the log.
             if (vrProvider.init()) {
                 playerCamera = new OpenVRStereoCamera(vrProvider);
+                /*
+                * The origin of OpenVR's coordinate system lies on the ground of the user. We have to move this origin
+                * such that the ground plane of the rendering system and the ground plane of the room the VR user is
+                * in match.
+                 */
+                //
+                vrProvider.getState().setGroundPlaneYOffset(-0.80f - context.get(Config.class).getPlayer().getEyeHeight());
```

Comment:
For the moment I've just removed that factor. I need to fire the rig up and see the relationship between the setting for eye height and where the ground plane ends up. All I remember is that there was an issue where the expected `setGroundPlaneYOffset(-eyeHeight);` did not work properly and I had to add a random offset to fix it.

(I'll update this in a bit).